File: YuanChengXu/1029/1029.cpp
#include <cstdio>
int main()
{
    int n;
    scanf("%d", &n);
    if(n > 50)
        return 0;
    float l[50] = {0}, sum = 0;
    for(int i = 0; i < n; i++){
        if(i == 0 || i == 1)
            l[i] = i + 2;
        else
            l[i] = l[i - 1] + l[i - 2];
        sum += l[i] / ((i + 1) * 2 + 1);
    }
    printf("%.2f", sum);
    return 0;
}

File: YuanChengXu/1029/1029.py
n = int(input())
sum = float(0)
C = []
if(n > 50):
    exit()
for i in range(0, n):
    if i == 0 or i == 1:
        C.append(i+2)
    else:
        C.append(C[i - 1] + C[i - 2])
    sum += float(C[i] / ((i + 1) * 2 + 1))
print("%.2f" % (sum), end='')
